src/test/java/com/microsoft/aad/adal4j/DeviceCodeFlowTest.java

```diff
@@ -0,0 +1,204 @@
+/*******************************************************************************
+ * Copyright © Microsoft Open Technologies, Inc.
```

Comment:
wrong license? seems like licenses needs to be validated for all files. I believe we have moved to the MIT license